test/testinit.js

```diff
@@ -157,7 +157,7 @@ TestManager = {
 TestManager.init( {
 	"jquery": {
 		urlTag: "jquery",
-		choices: "dev,min,git,3.x-git,3.x-git.min,3.4.1,3.3.1,3.2.1,3.1.1,3.0.0"
+		choices: "dev,min,git,git.min,3.x-git,3.x-git.min,3.4.1,3.3.1,3.2.1,3.1.1,3.0.0"
```

Comment:
Do you want to add `3.5.0` while you are here?

Reply:
Good point, will do.

Reply:
Done.